site/index.html

```
@@ -2290,6 +2290,12 @@ <h3>Browser-session setup</h3>
       <div class="community-plugin-author"><a href="https://github.com/AlexZeitler">@AlexZeitler</a></div>
     </div>
   </div>
+  <div class="community-plugin">
+      <div class="community-plugin-name"><a href="https://github.com/HANCORE-linux/cliamp-plugin-nightrider">cliamp-plugin-nightrider</a></div>
+      <div class="community-plugin-desc">Nightrider visualizer</div>
+      <div class="community-plugin-author"><a href="https://github.com/HANCORE-linux">@HANCORE-linux</a></div>
+    </div>
+  </div>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </div> ]

(tag-pair)